CMakeLists.txt

```diff
@@ -77,7 +77,7 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
 if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
-    add_compile_options(-fconcepts -Wno-literal-suffix)
```

Comment:
Either this, or the deletion below makes the build spammy again...

Reply:
@alimpfard Oops. I moved it incorrectly. Fix on the way